src/utils/move.js

```diff
@@ -0,0 +1,14 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+/**
+ * 0 에서 9까지 숫자 중 무작위로 뽑아 4 이상일 경우 전진합니다.
+ * 전진한 차(key)의 value는 +1이 됩니다.
+ * @param {Map} playersMap
+ */
+export default function move(playersMap) {
+  playersMap.forEach((val, key) => {
```

Comment:
```suggestion
  playersMap.forEach((count, carName) => {
```
`val, key`도 괜찮은 것 같은데 조금 더 명확하게 표현해도 좋을 것 같아요!

Reply:
오오! 그러게요! 감사합니당